include/criterion/assert.h

```diff
@@ -50,6 +50,13 @@
  *
  *****************************************************************************/
 # define cr_assert_fail(FormatString, ...) <internal>
+/**
+ * Skips the test
+ *
+ * The test is marked as skipped and the execution of the function is aborted.
+ *
+ *****************************************************************************/
+#define cr_skip_test(void) <internal>
```

Comment:
Don't specify a macro parameter, this makes `-pedantic` produce a warning when calling `cr_skip_test()`. Just put an empty pair of parenthesis.

Also, use a space between `#` and `define` here.